tests/operators/test_plots.py

```diff
@@ -113,7 +124,17 @@ def test_colorbar_map_levels_yaxis(cube, tmp_working_dir):
     cube.rename("toa_upward_shortwave_flux")
     cmap, levels, norm = plot._colorbar_map_levels(cube, axis="y")
     assert cmap is None
-    assert (levels == np.linspace(0, 350, 2)).all()
+    assert levels == [0, 500]
+    assert norm is None
+
+
+def test_colorbar_map_levels_yaxis_default(cube, tmp_working_dir):
+    """Test for variable without ymin, ymax."""
+    cube = iris.cube.Cube([np.arange(10)])
+    cube.rename("toa_upward_shortwave_flux_difference")
```

Comment:
```suggestion
    cube = iris.cube.Cube(np.arange(10), long_name="toa_upward_shortwave_flux_difference")
```
The cube can be created with the right name. We also don't need to wrap the data in an outer array.